app/components/task_list_items/assessment/recommended_application_type_component.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  module Assessment
+    class RecommendedApplicationTypeComponent < TaskListItems::BaseComponent
+      def initialize(planning_application:)
+        @planning_application = planning_application
+      end
+
+      private
+
+      attr_reader :planning_application
+
+      def link_text
+        "Choose application type"
+      end
+
+      def link_path
+        if planning_application.recommended_application_type.present?
+          planning_application_assessment_recommended_application_type_path(planning_application)
+        else
+          edit_planning_application_assessment_recommended_application_type_path(planning_application)
+        end
+      end
+
+      def status_tag_component
+        StatusTags::BaseComponent.new(status:)
+      end
+
```

Comment:
I believe this is unnecessary, since it's [inherited from the superclass](https://github.com/unboxed/bops/blob/main/app/components/task_list_items/base_component.rb#L11).

```suggestion
```

Reply:
That's true! I had status there at some point but then moved it to a different method.